.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -39,6 +39,7 @@ jobs:
           - coco-guest-components
           - firecracker
           - genpolicy
+          - kata-csi-directvolume
```

Comment:
Is this only supposed to be supported on amd64?

Reply:
I haven't tested this driver on other architectures, and since the CoCo CI only requires the amd64 tarball I only added this here. WDYT?

Reply:
Might do this in a separate PR if this requires more significant refactoring https://github.com/kata-containers/kata-containers/blob/b4da4b5e3b9b21048af9333b071235a57a3e9493/tools/packaging/static-build/tools/Dockerfile#L44

Reply:
Yeah, that makes sense, I hadn't appreciated that the tools were only build on x86. I guess we'll just need to skip s390x in the integration tests until we have this support in.

Reply:
Thanks for the heads-up! I was able to successfully build the `directvolplugin` binary on s390x. However, it seems there are several places that need to be refactored to support s390x here, which feels out of scope for this PR. As both of you mentioned, I’ll raise a separate PR to enable s390x build and test. ❤️

Reply:
Thank you @BbolroC! Please do reach out to let me know how I can support you - I definitely want to test this e2e on s390x as well if it's realistic. Will resolve this convo for now. 🙂